perfrunbook/utilities/capture_flamegraphs.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+help_msg() {
+ echo "usage: $0 oncpu|offcpu [time seconds|default 300]"
+}
+
+capture_on_cpu () {
+  perf record -a -g -k 1 -F99 -e cpu-clock:pppH -- sleep $1
+  perf inject -j -i perf.data -o perf.data.jit
+  perf script -f -i perf.data.jit > script.out
+  ./FlameGraph/stackcollapse-perf.pl --kernel --jit script.out > folded.out
```

Comment:
where is this expected to be run from.. perhaps you want to use `$(dirname $0)`here to find the directory the script is and construct a relative path?